constants.go

```diff
@@ -7,9 +7,13 @@ const (
 
 	EVENT_ADD_USER    string = "AddUserToGroup"
 	EVENT_REMOVE_USER string = "RemoveUserFromGroup"
+	GROUP_BASE_URL  string = "https://console.aws.amazon.com/iam/home?region=us-east-1#/groups/"
+	ROLE_BASE_URL   string = "https://console.aws.amazon.com/iam/home?region=us-east-1#/roles/"
+	USER_BASE_URL   string = "https://console.aws.amazon.com/iam/home?region=us-east-1#/users/"
 
-	ADD_USER_SLACK_MSG    string = ":red-light: User %s was added to group %s (IP: %s)"
-	REMOVE_USER_SLACK_MSG string = ":green-light: User %s was removed from group %s (IP: %s)"
+	ADD_USER_SLACK_MSG    string = ":red-light: User %s was added to group %s (*Arn:* %s *IP:* %s)"
+	REMOVE_USER_SLACK_MSG string = ":green-light: User %s was removed from group %s (*Arn:* %s *IP:* %s)"
```

Comment:
I think generally, this is a good approach, but I'm also wondering if the principal ID would be valuable to have.
Maybe we can keep an eye out for if this is sufficient (it probably is as we can use CloudTrail to narrow down to a principal if we need to)